docs/DESIGN.md

```diff
@@ -151,7 +151,7 @@ effort.
 vmm-reference                                                           \
     --memory size_mib=1024                                          \
     --vcpu num=1                                                 \
-    --kernel path=/path/to/vmlinux,himem_start=1024,cmdline="pci=off"   \
+    --kernel path=/path/to/vmlinux,kernel_load_addr=1024,cmdline="pci=off"   \
```

Comment:
If `kernel_load_addr` belongs to `kernel` argument, would it be enough to name it simply `load_addr`?

Reply:
In the same configuration we might also add the value for `cmdline_load_addr`. That's why I added kernel as well. It is a bit weird indeed.